src/components/ItemEditorPage/TopPanel/TopPanel.tsx

```diff
@@ -151,9 +156,12 @@ export default class TopPanel extends React.PureComponent<Props, State> {
     return isCommitteeMember && isReviewing && isConnected ? (
       <div className="TopPanel">
         <CollectionProvider id={selectedCollectionId}>
-          {({ collection, items: collectionItems, curation, isLoading }) =>
-            //TODO: Add logic to fetch the `itemCurations` array for the collection and pass it down to the `renderPage` call
-            !collection || isLoading ? <Loader size="small" active /> : this.renderPage(collection, collectionItems, curation, [])
+          {({ collection, items: collectionItems, itemCurations, curation, isLoading }) =>
```

Comment:
just in case you'd like to simplify this, items is just fine too!
```suggestion
          {({ collection, items, itemCurations, curation, isLoading }) =>
```